### DISCUSSION TITLE
How do you change the width?

### DISCUSSION THREAD

**Discussion by bubblydevs** [csandman/chakra-react-select#92] [Q&A] [answered]
How do you change the width? I tried reading everywhere and i still have not found it.

**Comment by csandman** [comment, marked as answer]
Sure, there are two easy ways you could do this.  The first is by setting a width on the `container` using the `chakraStyles` prop:

```js
<Select
  chakraStyles={{
    container: (provided) => ({
      ...provided,
      width: "300px"
    })
  }}
/>
```

The second would be to just add a container around the `Select` with a manual width set on that.

```js
<Box w="250px">
  <Select />
</Box>
```

Here's a CodeSandbox example with both: https://codesandbox.io/s/chakra-react-select-custom-width-8h04ww?file=/example.js

---

Let me know if either of these works for you!

**Reply by bubblydevs** [reply]
Thanks. It works

**Reply by csandman** [reply]
Glad I could help!
